uom_unece/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "Product UoM UNECE",
-    "version": "17.0.1.0.0",
+    "version": "18.0.1.0.0",
```

Comment:
Technically, version bump and contributors updates should land in the migration commit.
pre-commit commit is only to align with the new pre-commit config and commited with `--no-verify`.

Reply:
I updated commit